docs/en/observability/monitor-infra/host-metrics.asciidoc

```diff
@@ -0,0 +1,218 @@
+[[host-metrics]]
+= Host metrics
+
+Learn about key host metrics displayed in the {infrastructure-app}:
+
+* <<key-metrics-hosts,Hosts>>
+* <<key-metrics-cpu,CPU usage>>
+* <<key-metrics-memory,Memory>>
+* <<key-metrics-log,Log>>
+* <<key-metrics-network,Network>>
+* <<key-metrics-network,Disk>>
+
+
+//REVIEWERS: Note that the metrics are sorted alphabetically in the tables. This might
```

Comment:
Reviewers: Please note this comment and let me know if you have any concerns.

Reply:
+1 for alphabetical

Reply:
Thanks for chiming in @smith